benchexec/tablegenerator/__init__.py

```diff
@@ -1382,10 +1386,201 @@ def write_head_line(
         out.write("\n")
 
 
+def write_tex_command_table(
```

Comment:
Given that this part adds ~200 lines to an already long module, I guess a separate module is better.